packages/inspection-report/src/components/DamageReport/DamageReport.js

```diff
@@ -57,13 +58,25 @@ function DamageReport() {
   const { t } = useTranslation();
   const [tabIndex, setTabIndex] = useState(0);
   const [isPopUpVisible, setIsPopUpVisible] = useState(false);
+  const [isBigModalPopUpVisible, setIsBigModalPopUpVisible] = useState(false);
```

Comment:
rename this to `[isModalVisible, setIsModalVisible]`

Reply:
Done